resources/js/Components/ComponentsPraktikans/PraktikanAnno.jsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from "react";
+import notificationIcon from "../../../assets/Icon-Notification.svg";
+import CloseButton from "../CloseButton";
+
+export default function PraktikanAnnouncement() {
+    // State to manage popup visibility
+    const [isPopupOpen, setIsPopupOpen] = useState(false);
+    const isHaveAnnounce = true;
```

Comment:
Its better to rename this to `isAnnouncementExist` for better clarity

```suggestion
    const isAnnouncementExist = true;
```

Reply:
thanks for the suggestion, I will change it